src/msw/graphicsd2d.cpp

```diff
@@ -360,6 +360,221 @@ HRESULT WINAPI wxD3D11CreateDevice(
 }
 #endif
 
+#if wxUSE_PRIVATE_FONTS
+
+// This function is defined in src/msw/font.cpp.
+extern const wxArrayString& wxGetPrivateFontFileNames();
+
+namespace
+{
+    IDWriteFontCollection* gs_pPrivateFontCollection = NULL;
```

Comment:
Would it be possible to use `wxCOMPtr` for this one too to avoid explicit `Release()` calls?